cray/tests/test_unit/test_auth.py

```diff
@@ -94,7 +94,7 @@ def test_auth_load(cli_runner):
     username = opts['default']['username']
     hostname = opts['default']['hostname']
 
-    # token = {'test': 123, 'client_id': 'cray'}
+    # token = {'test': 123, 'client_id': 'shasta'}
```

Comment:
I thought we were not supposed to use "shasta".  I don't recall where I heard that, but that is why we changed from shasta 1.x to csm 1.x

Reply:
aah, yeah -- great comment.  this could change, but that'd be in keycloak -- but that's the client with all the recent multi-tenancy changes..